reservations/admin.py

```diff
@@ -94,9 +111,13 @@ class ReservationAdmin(admin.ModelAdmin):
             },
         ),
     ]
-    list_display = ("created_at", "first_name", "last_name")
+    list_display = ("created_at", "first_name", "last_name", "switch")
+    list_filter = (ReservationTypeFilter,)
     actions = ["export_reservations", "resend_reservation_confirmation"]
 
+    def switch(self, obj):
```

Comment:
maybe `switch_application` would be more descriptive? also, can we make this dynamic field translatable?

Reply:
Changed it to reservation type, is that ok?